graspologic/inference/latent_distribution_test.py

```diff
@@ -85,7 +85,7 @@ class LatentDistributionTest(BaseInference):
 
     pooled : bool (default=False)
         Ignored whenever the two graphs have the same number of vertices or
-        `size_correction` is set to False. In order to correct the adjacency
```

Comment:
i am confused by why all of the single backticks became double. did you have a reason for this?

Reply:
I changed it to double back ticks whenever it was referring to a parameter.

Reply:
In case it was unclear, in the sphinx documentation, single back ticks (by default) yield *emphasis* like `*emphasis*` in markdown.  Apparently this is an open issue of some controversy in SymPy: https://github.com/sympy/sympy/issues/13519.  I am happy to implement whichever version you think is best.

Reply:
@j1c @dwaynepryce any preferences or thoughts here?

Reply:
I'm pretty sure that scipy convention is that parameters are given a single tick, and other variables and equations are given a double tick.

Reply:
It looks better with double ticks imo.

Reply:
i think i agree that double looks better, also since that's what @pauladkisson did I will default to that unless @dwaynepryce or anyone else has strong feelings otherwise

Reply:
I think picking a standard (i.e. this PR) is most important, even in sklearn they are inconsistent (see https://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPClassifier.html#sklearn.neural_network.MLPClassifier for example)

Reply:
i agree with consistency being the goal here. and as long as we can't have it so that clicking on an attribute hyperrefs to where it is described (which we can't, right?) - i say we should go with whatever looks better.